frontend/public/metalkube/components/host/host.tsx

```diff
@@ -55,7 +62,21 @@ const HostRow = ({ obj: host }) => {
     },
   } = host;
 
-  const machineName = _.get(host, 'status.machineRef.name');
+  const machineName = getHostMachineName(host);
+  const machineResource = {
```

Comment:
you can use `getResource` from `web-ui-components` which has a lot of these fields prefilled

Reply:
yeah, I tried but problem is that `getResource` only outputs resource kind as `Model.kind` and this does not work for machines as machines are stored in redux store by key which is a result of `referenceForModel(MachineModel)`. So I had to specify the machineResource explicitly here.

I think the general goal is to keep all resources keyed using regerenceForModel in redux, so we'll have to update getResource function to handle it.